### DISCUSSION TITLE
Can you add the alien mantas in half life resurgence?

### DISCUSSION THREAD

**Discussion by trilobite2** [VJ-HLR-Developers/Half-Life-Resurgence#91] [Ideas]
In half life are 3 types of alien mantas: the surfrice tension manta, who deploys alien grunts and destroys ospreys; the other manta appears in ¨ Interlooper" and shoots a ray. Finally is Boss manta of decay.This manta appears in the hlr thank for reading.I put more ideas, good luck.

ATTE: Trilobite27

**Comment by Cpt-Hazama** [comment]
This isn't the place to put suggestions

**Reply by trilobite2** [reply]
In that case where i put suggestions?

**Reply by DrVrej** [reply]
> In that case where i put suggestions?

We don't take NPC requests, we have received over thousand different one, we don't need anymore lol

**Reply by trilobite2** [reply]
Ok are the same case of entities? Of all ways i wait you take in consideration.
